f8a_worker/dispatcher/flows/bayesianPackageFlow.yml

```diff
@@ -15,6 +15,8 @@
       edges:
         - from:
           to: 'InitPackageFlow'
+        - from: 'bayesianPackageAnalysisFlow'
+          to: 'UpstreamUpdateTask'
```

Comment:
Does this mean that upstream update task and the flow task will run simultaneously?

Reply:
Well, it wasn't my intention. I wanted to run it one after another. Does it mean that current configuration is bad?